assets/custom-icons/_data/custom-icons.json

```diff
@@ -285,6 +291,9 @@
       "slug": "standardnotes",
       "hex": "2173E6"
     },
+    {
+      "title": "Synology DSM"
```

Comment:
Ditto

Reply:
Should be better now :D
Could you please check again?